src/components/nutrition/label/nutrition-label.tsx

```diff
@@ -0,0 +1,210 @@
+    fontWeight: 100,
+    textAlign: "right",
+    margin: "3px 0 7px"
+}));
+
+const DailyValues = styled(Typography)(() => ({
+    lineHeight: "0.583rem",
+    fontSize: "0.583rem",
+    fontWeight: 100,
+    textAlign: "left",
+    margin: "3px 0 7px"
+}));
+
+function NutritionLabel({ backgroundColor, servingText, calories, caloriesFromFat, totalFat, saturatedFat, transFat,
+                          cholesterol, sodium, totalCarbohydrate, potassium, dietaryFiber, sugars, protein, vitamins,
+                          width, minWidth, recommendedValues: recValues, digits, multiplier }
+: {
+  backgroundColor?: string,
+  servingText: string,
+  calories: number,
+  caloriesFromFat: number,
+  totalFat: number,
+  saturatedFat: number,
+  transFat: number,
+  cholesterol: number,
+  sodium: number,
+  potassium: number,
+  totalCarbohydrate: number,
+  dietaryFiber: number,
+  sugars: number,
+  protein: number,
+  vitamins: Array<string>,
+  minWidth?: string,
+  width?: string,
+  digits?: number,
+  recommendedValues?: {
+    calories?: number,
+    caloriesFromFat?: number,
+    totalFat?: number,
+    saturatedFat?: number,
+    cholesterol?: number,
+    sodium?: number,
+    potassium?: number,
+    totalCarbohydrate?: number,
+    dietaryFiber?: number,
+    sugars?: number,
+    protein?: number,
+  },
+  multiplier?: number
+}) {
+
+    if (digits === undefined) {
+      digits = 1;
+    }
+
+    if (!multiplier || multiplier < 0) {
+      multiplier = 1;
+    }
+
+    return (
+        <LabelContainer backgroundColor={backgroundColor} width={width || "auto"} minWidth={minWidth || 0}>
+            <Title>Nutrition Facts</Title>
+            <ServingInfo>
+                {`Serving Size ${servingText}`}
+            </ServingInfo>
+
+            <SeparatorBar height={"0.583rem"} color={backgroundColor}/>
+
+            <AmountPerServing>
+                Amount Per Serving
+            </AmountPerServing>
+
+            <SeparatorBar height={"1px"} color={backgroundColor}/>
+
+            <CalorieRow>
+                <CaloriesLabel>
+                    Calories
+                </CaloriesLabel>
+                <CaloriesValue>
+                    {formatValue((calories || 0) * multiplier, 0)}
+                </CaloriesValue>
+                {caloriesFromFat > 0 && (
+                    <CaloriesFromFat>
+                        {`Calories from Fat ${(caloriesFromFat * multiplier).toFixed(digits)}`}
+                    </CaloriesFromFat>
+                )}
+            </CalorieRow>
+
+            <SeparatorBar height={"4px"} color={backgroundColor}/>
+
+            <PercentDailyValueLabel>
+                % Daily Value*
+            </PercentDailyValueLabel>
+
+            <SeparatorBar height={"1px"} color={backgroundColor}/>
+
+            <NutrientRow label={"Total Fat"} value={(totalFat || 0) * multiplier} unit="g" boldLabel
+                         recommended={(recValues?.totalFat || 78)} digits={digits} />
```

Comment:
Instead of defining them here, maybe do something like

```ts
// Somewhere like config/nutrition.ts
const FDA_RECOMMENDED_VALUES = {
  totalFat: 78,
  sodium: 2300
  // ...etc
};
```

```ts
// Merge base recommended values with custom macro goals
const recValues = {
   ...FDA_RECOMMENDED_VALUES,
  ...macroGoals
};
```

Reply:
Again, trying to avoid coupling